.github/workflows/node.js.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

Comment:
Should the result from running this action be already visible in this PR. Do I need to enable it for the repo or something?

Reply:
Nitpicking only shows that you have an eye for details. :) I will fix those issues asap.

Reply:
If you look at my fork you'll see that it's already working. So it should, too when it's merged into your master.

https://github.com/andreashouben/repoman/actions